### ISSUE TITLE
Cannot import data as PNG format

### ISSUE THREAD

**Issue by FlyingSamson** [WolframResearch/WolframLanguageForJupyter#119]
# What I observe
When I run, e.g.,
```
Solve[x==1]
```
in the Wolfram Language kernel I get no output and the following error messages in the jupyter log:
```
Import::fmterr: Cannot import data as PNG format.


Import::fmterr: Cannot import data as PNG format.


Import::fmterr: Cannot import data as PNG format.


General::stop: Further output of Import::fmterr

     will be suppressed during this calculation.


StringJoin::string: 

   String expected at position 2 in 

    <img alt="Output" src="data:image/png;base64,<>failedinBase64<>">.


StringJoin::string: 

   String expected at position 2 in 

    <div><img alt="Output" src="data:image/png;base64,<>failedinBase64<>"></div>.


Export::jsonstrictencoding: Expression StringJoin cannot be exported as JSON.
```

# What I expect:
To see the formula as embedded picture.

# What I already tried/figured out:
The latter one
```
StringJoin::string: 

   String expected at position 2 in 

    <img alt="Output" src="data:image/png;base64,<>failedinBase64<>">.
```
is due to a typo at this [line](https://github.com/WolframResearch/WolframLanguageForJupyter/blob/551351a70733866dd48146a8e1f8bae3a7789ffd/WolframLanguageForJupyter/Resources/OutputHandlingUtilities.wl#L327) where it should read failedInBase64 (capital "I").

The other ones
```
Import::fmterr: Cannot import data as PNG format.
``` 
are produces at this [call to ExportByteArray](https://github.com/WolframResearch/WolframLanguageForJupyter/blob/551351a70733866dd48146a8e1f8bae3a7789ffd/WolframLanguageForJupyter/Resources/OutputHandlingUtilities.wl#L279), but I was unable to figure out what the problem was. If I run above code in `wolfram` within the command line of the same docker container (in fact I used a terminal provided by the very same JupyterLab instance) I get the expected result
```
In[1]:= Solve[x==1]                                                                                                                                                                                                                                                     

Out[1]= {{x -> 1}}
```

Moreover all inputs resulting in plaintext output like `Print["Hello"]` or `1+1` do work as expected in the notebook displaying "Hello" and "2", respectively, so apparently something goes wrong with results that are returned as images.

How can I debug this issue? For a start how can I print/log what is actually passed to the `ExportByteArray` call? Is there some logging functionality build into the kernel that would allow to append messages to the jupyter log?

# My environment:
- `$Version` gives `12.2.0 for Linux ARM (32-bit) (April 7, 2021)`
- `jupyter version` gives
   ```
    IPython          : 7.30.0
    ipykernel        : 6.6.0
    ipywidgets       : 7.6.5
    jupyter_client   : 7.1.0
    jupyter_core     : 4.9.1
    jupyter_server   : 1.12.1
    jupyterlab       : 3.2.4
    nbclient         : 0.5.9
    nbconvert        : 6.3.0
    nbformat         : 5.1.3
    notebook         : 6.4.6
    qtconsole        : not installed
    traitlets        : 5.1.1
   ```
- `uname -a` gives `Linux aafe17fa30e2 5.4.0-1046-raspi #50-Ubuntu SMP PREEMPT Thu Oct 28 05:32:10 UTC 2021 aarch64 GNU/Linux`

**Comment by FlyingSamson** [comment]
So I just tried replacing the export formats with "JPEG" instead of "PNG" and this works. I presume this is an indicator that there is some problem with my installation of the wolfram engine and/or its dependencies rather than with the `Wolfram Language Kernel`?

**Comment by benfre** [comment]
I also find this problem on raspberry os, $Version -> "12.2.0 for Linux arm (32-bit)"

 To anyone want to know the "JPG" fix,  it's changing Format from "PNG" to "JPG" in call to `ExportByteArray[]` in function `toImageData[]` "Resources/OutputHanflingUtilities.wl".

**Comment by benfre** [comment]
I also find this problem on raspberry os, $Version -> "12.2.0 for Linux arm (32-bit)"

 To anyone want to know the "JPG" fix,  it's changing Format from "PNG" to "JPG" in call to `ExportByteArray[]` in function `toImageData[]` "Resources/OutputHanflingUtilities.wl".

**Comment by FlyingSamson** [comment]
I just posted this as a [question to the wolfram community](https://community.wolfram.com/groups/-/m/t/2465013).

**Comment by FlyingSamson** [comment]
This is now fixed with the latest update to Mathematica 13.0.1
